src/Parse/ParseUser.php

```diff
@@ -224,15 +218,30 @@ public static function logInWithTwitter(
                 'Cannot log in Twitter user without an auth token secret.'
             );
         }
+        $authData = [
+            'id'                => $id,
+            'screen_name'       => $screen_name,
+            'consumer_key'      => $consumer_key,
+            'consumer_secret'   => $consumer_secret,
+            'auth_token'        => $auth_token,
+            'auth_token_secret' => $auth_token_secret,
+        ];
+
+        return self::logInWith('twitter', $authData);
```

Comment:
Same thing here, all looks good but test coverage doesn't run through it all the way. We won't be able to provide actual credentials to test a 'pass' case but you can pass a fake `screen_name`, `consumer_key`, `auth_token`, etc. to get it to at least run through. Just so you know with a full request you'll be catching a ```ParseException``` with a message something like `'Twitter auth is invalid for this user.'`.